src/renderer/components/uielements/addressEllipsis/AddressEllipsis.tsx

```diff
@@ -39,29 +50,36 @@ export const AddressEllipsis: React.FC<Props> = (props): JSX.Element => {
 
   const measuredParent = useCallback(
     (node) => {
+      const prepareEllipse = () => prepEllipse(node, node.childNodes[0], node.childNodes[1])
       if (node !== null) {
-        window.addEventListener('resize', () => {
-          prepEllipse(node)
-        })
-        prepEllipse(node)
+        window.addEventListener('resize', prepareEllipse)
+        prepEllipse(node, node.childNodes[0], node.childNodes[1])
+      } else {
+        window.removeEventListener('resize', prepareEllipse)
```

Comment:
Is  `removeEventListener` really called when `AddressEllipsis` will be unmounted? Something like follow should do such thing:

```
useEffect(() => {
    return () => window.removeEventListener('resize', prepareEllipse)
}, [])
```